### PR TITLE
Added sending statistics about GPU

### PR DESCRIPTION
**Pull request by rokuz**

### CODE REVIEW

**Comment by rokuz** [comment]
@deathbaba PTAL

File: drape/support_manager.cpp [hunk 1]
```diff
@@ -12,6 +14,15 @@ void SupportManager::Init()
   string const version = GLFunctions::glGetString(gl_const::GLVersion);
   LOG(LINFO, ("Renderer =", renderer, "Version =", version));
 
+  // On Android the engine may be recreated. Here we guarantee that GPU info is sent once.
+  static bool gpuInfoSent = false;
+  if (!gpuInfoSent)
+  {
+    alohalytics::TStringMap details {{ "renderer", renderer }};
+    alohalytics::Stats::Instance().LogEvent("GPU", details);
```
Comment (deathbaba) [inline drape/support_manager.cpp]: You don't need <string, map> event here. LogEvent("GPU renderer", renderer) is enough.
Btw, GLFunctions::glGetString() never returns nullptr, right?

Comment (rokuz) [inline drape/support_manager.cpp]: Да, nullptr не будет

**Comment by rokuz** [comment]
@deathbaba PTAL

**Pull request merged**